engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
-- creation of tasks into the sprint board's backlog column
-- promotion of tasks into the sprint board's refined column
-- prioritization of task in the sprint board's refined column
+Backlog refinement meetings should aim to
+- Identify and remove tasks that are stale or obsolete
+- Identify and action tasks that are in the wrong status or priority on the board
+- Identify tasks that need further asynchronous refinement
 
-Tasks should tie back to initiatives, and the refined column should also function as a buffer of uncommitted work to pull from if needed.
+Tasks should tie back to initiatives or contract deliverables, and the refined column should also function as a buffer of uncommitted work to pull from if needed.
 
 Attendees of the Refinement meeting should include representatives from the following teams:
 - Engineering: for technical decision-making
 - Product: for product decision-making
-- Partnerships and Community Success: for community/grant decision-making.
+
+Product Managers at the meeting should ensure to come prepared to represent Business Development interests.
 
 (meetings:sprint-planning)=
-### 2. Iteration Planning
 
-The team conducts a synchronous iteration planning meeting for 60 minutes at the beginning of each iteration.
+### 2. Iteration Planning Guide
+
+#### 🗓️ Between Refinement and Planning
+
+A Product Manager will:
+
+- **Add deadlines to issues that need them**  
+  A Product Manager will ensure that any issue with a contractual obligation or expected delivery has a clear due date.
+
+- **Mark contract deliverables with dates**  
+  Use labels to easily identify Contract Deliverables.
+
+- **Assign preferred sprint-end dates**  
+  If a card is intended for the upcoming sprint, let a Product Manager know and they will mark it with this sprint’s end date.
+
+- **Organize in 'Up Next'**  
+  A Product Manager manages and sorts cards in the ‘Up Next’ column for visibility.
+
+The Team will:
+
+- **Review open-source contributions**  
+  Include relevant contributions and initiatives in planning discussions.
+
+- **Advocate for priority cards**  
+  Team members should speak up for issues they’re most invested in or want to lead. Reach out to a Product Manager in the days leading up to a sprint to discuss the priority of the issues you wish to champion:
+  - First check that the issues you want to suggest are not already in the Up Next column and dated for the upcoming sprint. 
+  - Send a message to the PM at least one day prior to the planning session with a list of issues you feel need to be included in the upcoming sprint.
+  - Please include as much context as needed for the PM to understand why these issues need to be in the upcoming sprint
+  - The PM will take your suggestions under advisement and make a best effort to include them, but cannot guarantee that to be the case as other priorities and capacity will need to be considered. 
+
+#### 🗺️ Roadmap & Initiative Review
```

Comment:
This too must happen before the meeting right? In that case I *think* the heading is one level off.